src/test/java/com/google/jenkins/plugins/storage/util/GoogleRobotCredentialsDescriptor.java

```diff
@@ -0,0 +1,79 @@
+package com.google.jenkins.plugins.storage.util;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.common.base.Strings;
+import com.google.jenkins.plugins.credentials.oauth.GoogleOAuth2ScopeRequirement;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotCredentials;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotCredentialsModule;
+import com.google.jenkins.plugins.credentials.oauth.Messages;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.util.FormValidation;
+import java.security.GeneralSecurityException;
+import org.kohsuke.stapler.QueryParameter;
+
+/**
+ * Dev Memo:
+ * In earlier versions, {@code AbstractGoogleRobotCredentialsDescriptor} was a standalone
+ * abstract class responsible for handling descriptors for Google robot account credential extensions.
+ * Now, {@code AbstractGoogleRobotCredentialsDescriptor} has been restructured and
+ * integrated as a static inner class within {@code GoogleRobotCredentials}.
+ *
+ * This change affects how the descriptor is accessed and used within the codebase. No major
+ * changes in the API usage are expected, but minor adjustments might be necessary in how the class
+ * is instantiated or accessed
+ */
+public class GoogleRobotCredentialsDescriptor extends GoogleRobotCredentials {
```

Comment:
Why are you copy&pasting this class here?
No tests in this plugin should depend on a protected inner class from another plugin.

Reply:
Tests in this plugin were using `AbstractGoogleRobotCredentialsDescriptor` but this descriptor class was [deleted](https://github.com/jenkinsci/google-oauth-plugin/pull/201/files#diff-4caf35987ce7b76e4d3e7c40fceffcf5658d1ac405be66a2a8feccebe52d68e7) from google-oauth-plugin and put in [GoogleRobotCredentials](https://github.com/jenkinsci/google-oauth-plugin/blob/develop/src/main/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotCredentials.java#L169)

Reply:
Yeah, but why do this tests need `GoogleRobotCredentialsDescriptor`?

Reply:
This [test](https://github.com/jenkinsci/google-storage-plugin/pull/277/commits/cb891b873eb5d9e622ad07b9f6dade4c471d3ead#diff-ff6efee22c1f22c7a1f1aafa5e15995fd91b9c871818229160fa4c85d807e21aR57) was failing for me, but it seems to be some local issue and now its working fine. 
I've [removed](https://github.com/jenkinsci/google-storage-plugin/pull/277/commits/cb891b873eb5d9e622ad07b9f6dade4c471d3ead) the descriptor test class, as its not necessary to have it in the tests